certification/internal/policy/container/has_unique_tag.go

```diff
@@ -24,10 +24,22 @@ type hasUniqueTagCheck struct {
 }
 
 func (p *hasUniqueTagCheck) Validate(ctx context.Context, imgRef certification.ImageReference) (bool, error) {
-	tags, err := p.getDataToValidate(ctx, fmt.Sprintf("%s/%s", imgRef.ImageRegistry, imgRef.ImageRepository))
+	imgRepo := fmt.Sprintf("%s/%s", imgRef.ImageRegistry, imgRef.ImageRepository)
+	tags, err := p.getDataToValidate(ctx, imgRepo)
 	if err != nil {
-		return false, fmt.Errorf("could not get image tags: %v", err)
+		err = fmt.Errorf("Error: failed to get tags list for %s: %w", imgRepo, err)
+		return false, err
```

Comment:
Two things (and this goes for line 38 as well)

1. Don't bother reassigning err. Just use fmt.Errorf in the return statement directly.
2. Use %v instead of %w. We don't want to leak the actual error from here.